### PR TITLE
Feature/ethusd rates

### PR DESCRIPTION
**Pull request by mkryna**

### CODE REVIEW

File: contracts/JavvyCrowdsale.sol [hunk 1]
```diff
@@ -10,6 +11,7 @@ import "./JavvyToken.sol";
 contract JavvyCrowdsale is RefundableCrowdsale, CappedCrowdsale, Config {
     uint256 public icoStartTime;
     address public bonusAddress;
+    uint private USDETHRate;
```
Comment (KJagiela) [inline contracts/JavvyCrowdsale.sol]: Better use `uint256` - it's the same but more explicit

File: contracts/JavvyCrowdsale.sol [hunk 2]
```diff
@@ -172,4 +177,15 @@ contract JavvyCrowdsale is RefundableCrowdsale, CappedCrowdsale, Config {
         _deliverTokens(wallet, token.getRemainingSaleTokens(), this);
         _deliverTokens(wallet, token.getRemainingBonusTokens(), bonusAddress);
     }
+
+    function setUSDETHRate(uint _USDETHRate) public
```
Comment (KJagiela) [inline contracts/JavvyCrowdsale.sol]: Also in this function - `uint256`

File: test/test_crowdsale.js [hunk 3]
```diff
@@ -145,6 +146,7 @@ contract('Crowdsale', function(accounts) {
     it('should different periods of ICO work', async () => {
         let amount = utils.ether(1);
         let preIcoTokens = amount * (crowdsaleUtils.rate + crowdsaleUtils.bonusRate);
+        let preIcoDoubleTokens = amount * (2*crowdsaleUtils.rate + crowdsaleUtils.bonusRate);
```
Comment (KJagiela) [inline test/test_crowdsale.js]: Why's that?

File: test/utils/crowdsale.js [hunk 4]
```diff
@@ -5,14 +5,15 @@ const utils = require('./utils');
 
 class Crowdsale {
     constructor() {
-        this.rate = 2625;
-        this.bonusRate = 656.25;
+        this.rate = 4687;
```
Comment (KJagiela) [inline test/utils/crowdsale.js]: Why this rate instead of the real one?

Comment (mkryna) [reply test/utils/crowdsale.js]: Good catch. Originally I wanted to change rate according to USDETHRate and adjusted normal rate to be correct. Now I should change it back . On it!

File: test/test_crowdsale.js (hunk 3 above)
Comment (mkryna) [reply test/test_crowdsale.js]: Previously when it was preIcoTokens + icoTokens which is basically the same as current preIcoDoubleTokens, math did not check out(there was 1 Wei extra), no time to check why the math is incorrect the old way so it stays like that for now

**Pull request merged**